packages/dev/s2-icon-builder/.parcelrc

```diff
@@ -0,0 +1,17 @@
+{
+  "namers": ["parcel-namer-icon"],
+  "bundler": "@parcel/bundler-library",
+  "transformers": {
+    "*.svg": ["parcel-transformer-icon"],
```

Comment:
we'll need to publish these plugins too I guess. Maybe we should rename them so they are under a scope, e.g. `@react-spectrum/parcel-transformer-icon`